app/controllers/dashboard/contributors/show.js

```diff
@@ -1,12 +1,17 @@
 import Controller from '@ember/controller';
 import { computed } from '@ember/object';
+import config from 'kredits-web/config/environment';
 
 export default Controller.extend({
 
   roleName: computed('model.isCore', 'totalKreditsEarned', function() {
     if (this.model.isCore) return 'Core Contributor';
     if (this.model.totalKreditsEarned <= 5000) return 'Newcomer';
     return 'Contributor';
-  })
+  }),
+
+  ipfsGatewayUrl: computed(function() {
+    return config.ipfs.gatewayUrl;
+  }).volatile()
```

Comment:
Why does this property need to be volatile?

Reply:
No, marking a property as volatile just deactivates caching and makes it recalculate on every access. It also prevents property notifications.

The `volatile()` modifier [got deprecated recently](https://deprecations.emberjs.com/v3.x/#toc_computed-property-volatile). So I think just using an empty string as the dependency would be better here.